dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/analysis/PhenotypeAnalyzer.java

```diff
@@ -58,7 +58,7 @@
     // Here we eliminate duplicates and impose ordering (needed for reading the response items).
     val setIds = ImmutableSet.copyOf(entitySetIds).asList();
 
-    val multiResponse = donorRepository.performPhenotypeAnalysisMultiSearch(setIds);
+    val multiResponse = donorRepository.entitySetMultiSearch(setIds);
```

Comment:
Here's an example of how we did this for EA:

https://github.com/icgc-dcc/dcc-portal/blob/develop/dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/analysis/EnrichmentAnalyzer.java#L369-L381